eng/Version.Details.xml

```diff
@@ -52,9 +52,13 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>d099f075e45d2aa6007a22b71b45a08758559f80</Sha>
     </Dependency>
-    <Dependency Name="System.Text.Json" Version="7.0.3">
-      <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>5b20af47d99620150c53eaf5db8636fdf730b126</Sha>
+    <Dependency Name="System.Text.Json" Version="8.0.4">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
+      <Sha>2aade6beb02ea367fd97c4070a4198802fe61c03</Sha>
+    </Dependency>
+    <Dependency Name="System.Formats.Asn1" Version="8.0.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
+      <Sha>2aade6beb02ea367fd97c4070a4198802fe61c03</Sha>
```

Comment:
```xml
<Dependency Name="System.Collections.Immutable" Version="7.0.0">
      <Uri>https://github.com/dotnet/runtime</Uri>
      <Sha>d099f075e45d2aa6007a22b71b45a08758559f80</Sha>
</Dependency>
```
(above)  needs to be updated as well.

Reply:
The sha will have to be updated as well. I suggest using `darc` to update these dependencies since the tool will take care of that for you.

Reply:
Updated uri and sha for System.Collections.Immutable 8.0.0. Please help to take another look.